vercel.json

```
@@ -12,52 +12,52 @@
     },
     {
       "source": "/(proving/challenge-manager/?)",
-      "destination": "/how-arbitrum-works/fraud-proofs/challenge-manager",
+      "destination": "/how-arbitrum-works/07-interactive-fraud-proofs",
```

```suggestion
      "destination": "/how-arbitrum-works/interactive-fraud-proofs",
```
Redirection destinations must be specified as URLs, meaning that they can't have the internal number of the file.

---

Changed entries, also added proper syntax formatting.